### ISSUE TITLE
txt2AL transform: error when trying to create new page action structure on comment sheet page

### ISSUE THREAD

**Issue by Srbaxter52** [Microsoft/AL#2843]
I am trying to transform 7 pages where the only change we did was to add the Page Action choices.  When I run the delta for the object through the txt2AL conversion tool, it errors out and gives the message: "Object reference not set to an instance of an object."

Here is a copy of the changes to the object.

[PAG99000838.txt](https://github.com/Microsoft/AL/files/2180357/PAG99000838.txt)

We have a subscriber elsewhere that will run code.

**Comment by AskeHolst** [comment]
Just a guess: there is an action with the name "Co&mments" try changing it to "Comments".

**Comment by Srbaxter52** [comment]
There are no Page Actions on the base page.  I am trying to create the menu.

**Comment by qutreson** [comment]
Hi @Srbaxter52, which version of the server are you using?
I tried reproducing your issue on our latest master and I didn't get the error you described.

Here is what I am getting this as result:
```
pageextension 70000005 pageextension70000005 extends "Prod. Order Comment Sheet" 
{
    actions
    {
        addfirst(processing)
        {
            group("Co&mments")
            {
                Caption = 'Co&mments';
            }
            action(WordWrap)
            {
                Caption = 'Word Wrap Comments';
                Image = Comment;
                Promoted = true;
                PromotedCategory = Process;
                PromotedIsBig = true;
                ToolTip = 'Open Word Wrap Editor';
            }
        }
    }
}
```

It seems that we have a bug here because the new action is not nested in the group.

**Comment by Srbaxter52** [comment]
My container is running the latest bcsandbox as of 12 days ago. I should be pulling the txt2AL from the container to do the transform as I do not have NAV installed locally.

Thank you,

Sam Baxter
Cost Control Software
ph: (317) 846 6025

**Comment by Srbaxter52** [comment]
I just recreated my container.  It did download new items for the image.  The current NAV Version for the container is: 12.2.22932.23097-W1.  My image is: microsoft/bcsandbox:latest.  I reran my process and it still errors on the same 7 objects where I am having it create the page action menu.

I ran the same objects through a manual conversion using the txt2AL from the release of NAV 2018 and it worked manually. I will need to take a look at my tool that was doing this and see what went wrong there.


Thank you,

Sam Baxter
Cost Control Software
ph: (317) 846 6025
